newrelic-agent/src/main/java/com/newrelic/agent/config/AgentConfigFactory.java

```diff
@@ -229,7 +231,22 @@ public static void mergeServerData(Map<String, Object> settings, Map<String, Obj
         }
 
         // Copy "event_harvest_config" over from the serverData since it doesn't live in the "agent_config" subsection (it's a top level property from the collector)
-        addServerProp(AgentConfigFactory.EVENT_HARVEST_CONFIG, serverData.get(EVENT_HARVEST_CONFIG), settings);
+        Object eventHarvestConfig = serverData.get(EVENT_HARVEST_CONFIG);
+        addServerProp(AgentConfigFactory.EVENT_HARVEST_CONFIG, eventHarvestConfig, settings);
+
+        // when log forwarding is disabled account wide, the backend will inform the agent by setting the harvest limit to 0
+        if (eventHarvestConfig instanceof Map) {
+            Object harvestLimits = ((Map<?,?>) eventHarvestConfig).get(HarvestServiceImpl.HARVEST_LIMITS);
+            if (harvestLimits instanceof Map) {
+                Object logLimit = ((Map<?,?>) harvestLimits).get(CollectorMethods.LOG_EVENT_DATA);
+                if (logLimit instanceof Number && ((Number) logLimit).intValue() == 0) {
+                    String loggingForwardingEnabled = AgentConfigImpl.APPLICATION_LOGGING + "." + ApplicationLoggingConfigImpl.FORWARDING + "." +
+                            ApplicationLoggingForwardingConfig.ENABLED;
+                    addServerProp(loggingForwardingEnabled, false, settings);
+                }
```

Comment:
I had logging when this happens, but there is another log statement that kinda notifies us that log forwarding is turned off.